projects/i18n/tools/language.service.ts

```diff
@@ -0,0 +1,19 @@
+import {inject, Injectable} from '@angular/core';
+import {LOCAL_STORAGE} from '@ng-web-apis/common';
+import {Language} from '@taiga-ui/i18n';
+import {TUI_I18N_STORAGE_KEY, TUI_STORED_LANGUAGE} from '@taiga-ui/i18n/tools';
+import {ReplaySubject} from 'rxjs';
+@Injectable()
+export class TuiLanguageService extends ReplaySubject<Language> {
+    storedLanguage = inject(TUI_STORED_LANGUAGE);
```

Comment:
We do not write `inject` like that but rather add them with `@Inject` in constructor